hazelcast-jet-core/src/main/resources/client-protocol-definition/Jet.yaml

```diff
@@ -21,9 +21,14 @@ methods:
           doc: ''
         - name: jobConfig
           type: Data
-          nullable: false
+          nullable: true
           since: 2.0
           doc: ''
+        - name: isLightJob
+          type: boolean
+          nullable: false
+          since: 2.3
```

Comment:
Also, a micro optimization but, if we do not care for compatibility at all, we can set the since version of this parameter to `2.0`, to get rid of the extra if check in the decode part of the codecs and isIsLightJobExists flags of the request parameters objects.

Note that, doing this means that no old reader will be able to read messages from new writers, and no new readers will be able to read messages from old writers.

Reply:
If we stick with 2.3, will the error messages be more informative maybe?

Reply:
I am not sure about the clarity of the error messages in that case but I am okay with leaving things as it is. 

With `2.3` since version, the only problem old readers will have is when the `jobConfig` is null. With `2.0` version, they probably will not be able to read `jobConfig` properly at all, even if it is not null.

I have approved the PR